File: db/csharp/SalesManagement.Domain/Models/CompanyCategory.cs
namespace SalesManagement.Domain.Models
{
    /// <summary>
    /// 取引先分類マスタ
    /// </summary>
    public class CompanyCategory
    {
        public string CategoryTypeCode { get; set; } = string.Empty;
        public string CategoryCode { get; set; } = string.Empty;
        public string CategoryName { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public string CreatedBy { get; set; } = string.Empty;
        public DateTime UpdatedAt { get; set; }
        public string UpdatedBy { get; set; } = string.Empty;
    }
}

File: db/fsharp/SalesManagement.Domain/Models/CompanyCategory.fs
namespace SalesManagement.Domain.Models

open System

/// <summary>
/// 取引先分類マスタ（分類種別ごとの分類を定義）
/// </summary>
[<CLIMutable>]
type CompanyCategory = {
    CategoryTypeCode: string
    CategoryCode: string
    CategoryName: string
    CreatedAt: DateTime
    CreatedBy: string
    UpdatedAt: DateTime
    UpdatedBy: string
}
